File: ClientApp/src/app/models/filter.ts
//export class Filter {
//  initialDate: string = new Date(Date.now() - 2 * 24 * 60 * 60 * 1000).toISOString();
//  endDate: string = new Date().toISOString();
//  includeCancelled: boolean = true;
//  includeValid: boolean = true;
//  includeAvulso: boolean = true;
//  includeNotAvulso: boolean = true;
//}

export interface Filter {
  initialDate: string
  endDate: string
  includeCancelled: boolean
  includeValid: boolean
  includeAvulso: boolean
  includeNotAvulso: boolean
}

File: Models/Filter.cs
using System;

namespace target_teste.Models
{
    public class Filter
    {
        public DateTime InitialDate {  get; set; }
        public DateTime EndDate { get; set; }
        public bool IncludeCancelled { get; set; }
        public bool IncludeValid { get; set; }
        public bool IncludeAvulso { get; set; }
        public bool IncludeNotAvulso { get; set; }
    }
}
